README.md

```diff
@@ -3,9 +3,18 @@ WebLogic Monitoring Exporter
 
 [![Build Status](https://travis-ci.org/oracle/weblogic-monitoring-exporter.svg?branch=master)](https://travis-ci.org/oracle/weblogic-monitoring-exporter)
 
-The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
-It is available both as a [web application](#web-application) to be deployed to the WebLogic Server (WLS) instance, version 12.2.1 or later, from which you want to get metrics, 
-and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
+The WebLogic Monitoring Exporter is a [Prometheus](http://prometheus.io)-compatible exporter of metrics from 
+WebLogic Server (WLS) instances, which it obtains via the 
+[WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later. 
+Metrics are selected using a [YAML configuration file](#Configuration).
+
+The exporter is available in two forms:
+ - a [web application](#web-application) that you deploy to the managed server from which metrics are to be extracted. 
```

Comment:
i assume one might want admin server metrics, right?

if so, "managed server" --> "WebLogic server"

Reply:
"managed server" --> "WebLogic Server"

Reply:
No, this is only for managed servers.

Reply:
There is an ongoing discussion about this in slack.

Reply:
I have change it to say server instance. admin/managed/independent are not relevant.